birdhouse/env.local.example

```diff
@@ -204,6 +204,22 @@ export GEOSERVER_ADMIN_PASSWORD="${__DEFAULT__GEOSERVER_ADMIN_PASSWORD}"
 # (note: if using 'BIRDHOUSE_DATA_PERSIST_ROOT', it must be defined earlier, either in this file or from 'default.env')
 #export BIRDHOUSE_LOGROTATE_DATA_DIR='${BIRDHOUSE_DATA_PERSIST_ROOT}/logrotate'
 
+# These variables configure the scheduler-job-clean_old_files component
+#
+# Delete THREDDS log files older than X days (e.g. X=20):
+#export THREDDS_DELETE_FILES_OLDER_THAN_DAYS=20
+#
+# Delete WPS output files older than X days (e.g. X=90):
+#export WPS_OUTPUTS_DELETE_FILES_OLDER_THAN_DAYS=90
+# 
+# Set cron schedule for the clean old files job (how often the job runs).
+# By default it runs weekly on Sunday at 2:05 am:
+#export SCHEDULER_JOB_CLEAN_OLD_FILES_FREQUENCY="5 2 * * 0"
```

Comment:
I would avoid anything close to 2 and 3 AM to avoid problems on Daylight saving time switch twice a year.  Make it 1:05 am or 4:05 am to be sure.